sass/sass_wrapper.js

```diff
@@ -13,11 +13,44 @@
 const {debug, runAsWorker, runWorkerLoop} = require('@bazel/worker');
 const sass = require('sass');
 const fs = require('fs');
+const minimist = require('minimist');
 
 const args = process.argv.slice(2);
 if (runAsWorker(args)) {
   debug('Starting Sass compiler persistent worker...');
-  runWorkerLoop(args => sass.cli_pkg_main_0_(args));
```

Comment:
What's the goal of this change? It adds a lot of complexity and requires us to update multiple places any time we want to change which arguments are passed in to the compiler.

Reply:
It's to address #96. The issue is that the Dart _Future is not directly translatable to JS promises, so the worker loop doesn't have any idea when the compilation is actually done.

I couldn't find a way to wrap a Future in a Promise on the JS side; it may have to be done in Dart.

One other option could be to try and farm out to an execFile so we can just pass the arguments directly.

Reply:
I'd rather fix the root cause by adding support to http://github.com/google/dart_cli_pkg for wrapping futures in Promises than make this package harder to maintain going forward.

Reply:
I'm guessing that's https://github.com/google/dart_cli_pkg/issues/50? I'm happy to take a look there; let me shelve this for now then.